wagtail/admin/templates/wagtailadmin/home/upgrade_notification.html

```diff
@@ -1,5 +1,19 @@
-{% load wagtailcore_tags wagtailadmin_tags %}
+{% load i18n wagtailcore_tags wagtailadmin_tags %}
 
-<div data-upgrade data-wagtail-version="{% wagtail_version %}" class="panel nice-padding panel-upgrade-notification" style="display:none">
-    <div class="help-block help-warning">{% icon name='warning' %}Wagtail upgrade available. Your version: <strong>{% wagtail_version %}</strong>. New version: <strong data-upgrade-version></strong>. <a data-upgrade-link href="">Read the release notes.</a></div>
+{% wagtail_version as version %}
+<div
+    class="panel nice-padding panel-upgrade-notification w-hidden"
+    data-controller="w-upgrade-notification"
+    data-w-upgrade-notification-hidden-class="w-hidden"
+    data-w-upgrade-notification-url-value="https://releases.wagtail.org/latest.txt"
+    data-w-upgrade-notification-version-value="{{ version }}"
+>
+    <div class="help-block help-warning">
+        {% icon name='warning' %}
+        {% blocktrans trimmed %}
+            Wagtail upgrade available. Your version: <strong>{{ version }}</strong>.
+            New version: <strong data-w-upgrade-notification-target="latestVersion"></strong>.
+        {% endblocktrans %}
+        <a href="" data-w-upgrade-notification-target="link">{% translate "Read the release notes." %}</a>
```

Comment:
This should just be `trans`, right?

```suggestion
        <a href="" data-w-upgrade-notification-target="link">{% trans "Read the release notes." %}</a>
```

Reply:
Thanks, yeah it is up until Django 4.0 :) my bad, will use the `trans`.